### ISSUE TITLE
missing bitcoin-cli

### ISSUE THREAD

**Issue by jsarenik** [lightningamp/lightning_ndk#5]
https://github.com/lightningamp/lamp/issues/92

The issue is here. In the release archives of `0.9.1` version, there is `bitcoin-cli`. In `0.9.3` there is not.

@lvaccaro @vincenzopalazzo please have a look

**Comment by lvaccaro** [comment]
Thanks to report it. It could be a good time also to update the bitcoind binaries

**Comment by jsarenik** [comment]
I was able to manually install `bitcoin-cli` from the `0.9.1` archive (since I already needed `lightningd` version `0.9.3` because its internal databases were upgraded once I ran it).

The reason for running LAMP against local network `bitcoind` node is to make updates faster. When my LAMP becomes 30k blocks behind, it is almost impossible to sync over Esplora plugin.

**Comment by lvaccaro** [comment]
I agree, it looks quite unusable performing the sync when you miss so much blocks without an external service (other than esplora). An external font of data is a trust tradeoff but it could be necessary for these scenarios.

**Comment by jsarenik** [comment]
Is this still alive?
